packages/frontend/core/src/pages/subscribe.tsx

```diff
@@ -49,7 +63,7 @@ export const Component = () => {
           setMessage('Checking subscription status...');
           await subscriptionService.subscription.waitForRevalidation(signal);
           const subscribed =
-            plan?.toLowerCase() === 'ai'
+            receivedPlan === 'ai'
               ? !!subscriptionService.subscription.ai$.value
               : !!subscriptionService.subscription.pro$.value;
           if (!subscribed) {
```

Comment:
should allow user checking out for lifetime sub even has yearly/monthly subscription before